examples/notebooks/use-cases/BBP KG Forge retrieval.ipynb

```diff
@@ -78,26 +78,26 @@
      "output_type": "stream",
      "text": [
       "Execution 0  using:  id inside:  same bucket  - Cross bucket: True  - Retrieve source: True\n",
-      "rev 3\n",
+      "rev 3 https://bbp.neuroshapes.org https://bbp.epfl.ch/nexus/v1/resources/dke/kgforge/_/20fbc97a-fb26-43ff-8093-9136aab25dff\n",
       "______________________\n",
       "Execution 1  using:  self inside:  same bucket  - Cross bucket: True  - Retrieve source: True\n",
-      "rev 3\n",
+      "rev 3 https://bbp.neuroshapes.org https://bbp.epfl.ch/nexus/v1/resources/dke/kgforge/_/20fbc97a-fb26-43ff-8093-9136aab25dff\n",
       "______________________\n",
       "Execution 2  using:  id inside:  other bucket  - Cross bucket: True  - Retrieve source: True\n",
-      "rev 3\n",
+      "rev 3 https://bbp.neuroshapes.org http://purl.obolibrary.org/obo/GO_0038048\n",
       "______________________\n",
       "Execution 3  using:  self inside:  other bucket  - Cross bucket: True  - Retrieve source: True\n",
-      "rev 3\n",
+      "rev 3 https://bbp.neuroshapes.org http://purl.obolibrary.org/obo/GO_0038048\n",
       "______________________\n",
       "Execution 4  using:  id inside:  same bucket  - Cross bucket: False  - Retrieve source: True\n",
-      "rev 3\n",
+      "rev 3 https://bbp.neuroshapes.org https://bbp.epfl.ch/nexus/v1/resources/dke/kgforge/_/20fbc97a-fb26-43ff-8093-9136aab25dff\n",
       "______________________\n",
       "Execution 5  using:  self inside:  same bucket  - Cross bucket: False  - Retrieve source: True\n",
-      "rev 3\n",
+      "rev 3 https://bbp.neuroshapes.org https://bbp.epfl.ch/nexus/v1/resources/dke/kgforge/_/20fbc97a-fb26-43ff-8093-9136aab25dff\n",
       "______________________\n",
       "Execution 6  using:  id inside:  other bucket  - Cross bucket: False  - Retrieve source: True\n",
       "<action> catch_http_error\n",
-      "<error> RetrievalError: resource 'http://purl.obolibrary.org/obo/GO_0038048' not found in project 'dke/kgforge'\n",
+      "<error> RetrievalError: 404 Client Error: Not Found for url: https://bbp.epfl.ch/nexus/v1/resources/dke/kgforge/_/http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FGO_0038048\n",
```

Comment:
the error messages using id and self are different, is it possible to make them the same?